<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RoomParametersInputForm</class>
 <widget class="QDialog" name="RoomParametersInputForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>314</width>
    <height>194</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Введите параметры комнаты</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="lengthLabel">
       <property name="text">
        <string>Длинна( ось х):</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="lengthInput"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="widthLabel">
       <property name="text">
        <string>Ширина(ось z):</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="widthInput"/>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="heightLabel">
       <property name="text">
        <string>Высота(ось y):</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="heightInput"/>
     </item>
     <item row="4" column="1">
      <widget class="QPushButton" name="OKBtn">
       <property name="text">
        <string>Ок</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLineEdit" name="stepInput"/>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="stepLabel">
       <property name="text">
        <string>Шаг</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
